natural-language-understanding/src/main/java/com/ibm/watson/developer_cloud/natural_language_understanding/v1/model/Model.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.natural_language_understanding.v1.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * .
+ */
+public class Model extends GenericModel {
+
+  /** Shows as available if the model is ready for use. */
+  private String status;
+  /** Unique model ID. */
+  @SerializedName("model_id")
+  private String modelID;
```

Comment:
I think the convention in the Java SDK is to use "Id" rather than "ID" (this differs from Swift by the way).  So to adhere to that convention, this variable and its associated getters/setters should use "Id".